packages/builder/src/reducers/projects.ts

```diff
@@ -35,6 +35,10 @@ export type Application = {
   status: AppStatus;
   inReview: boolean;
   chainId: ChainId;
+  metaPtr?: {
```

Comment:
can the metaPtr ever be null? if not, remove the optional `?` please

Reply:
Yes metaPtr can be null in some cases.